docs/www/guide-wasm-elastic.md

```diff
@@ -0,0 +1,189 @@
+# Redpanda wasm function + Elastic Search
+One of the richest wasm function case is indexing information to [ElasticSeach](https://www.elastic.co/elasticsearch) from Redpanda's records. for this we need to requirements:
+
+1. Redpanda > v21.1.5
+2. ElasticSearch service, in this case we use the [docker image](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html), in localhost:9200
+
+## Enable wasm on Redpanda
+Add a `coproc_enable: true` in redpanda.yaml
```

Comment:
```suggestion
Add this line `coproc_enable: true` to your redpanda.yaml file, under the `redpanda` section
```

Reply:
You'll also need to say that developer mode should be set to true as well

Reply:
Suggestion to the suggestion:
```diff
- Add this line `coproc_enable: true` to your redpanda.yaml file, under the `redpanda` section
+ Add `coproc_enable: true` in your redpanda.yaml file, under the `redpanda` section. You can do this manually or by running `sudo rpk redpanda config set 'redpanda.coproc_enable' true`.
```

Reply:
Also, idk if it's too late, but the (ambiguous) convention I see among the current config fields is that it should be either

- `enable_xyz`
- `xyz_enabled`

https://vectorized.io/docs/advanced-configuration/

Reply:
Rpk to set the developer mode ON: `sudo rpk redpanda mode dev`

Reply:
Done.

Reply:
> Also, idk if it's too late, but the (ambiguous) convention I see among the current config fields is that it should be either
> 
> * `enable_xyz`
> * `xyz_enabled`
> 
> https://vectorized.io/docs/advanced-configuration/

should we create an issue for it?